backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -67,23 +66,17 @@ class ZaaktypenGetParametersQuery(OBPagedQueryParams, kw_only=True, rename="came
     ] = UNSET
 
 
-class ZaakTypeSummary(Struct, kw_only=True):
+class ZaakTypeSummary(VersionedResourceSummary, kw_only=True, rename="camel"):
     url: str
     identificatie: str
     omschrijving: str
-    # Actief ja/nee: calculated
-    actief: bool | UnsetType = UNSET
-    einde_geldigheid: datetime.date | None = field(default=None, name="eindeGeldigheid")
     # str, because VertrouwelijkheidaanduidingEnum does not contain "" but OZ does
-    # XXX: the "" is actually a fault in the fixtures!
     vertrouwelijkheidaanduiding: str
     versiedatum: datetime.date
-
-    def __post_init__(self):
-        self.actief = (
-            self.einde_geldigheid is None
-            or datetime.date.today() < self.einde_geldigheid
-        )
+    # Actief true/false: calculated for ja/nee in the frontend
+    actief: bool | UnsetType = UNSET
+    einde_geldigheid: datetime.date | None = None
+    concept: bool | UnsetType = UNSET
```

Comment:
A shame we need to repeat these from the parent. I think it's because those fields in the mro are member descriptors of the parent and not this child. I might peek upstream if I ever get bored.